src/track.jsx

```diff
@@ -16,19 +16,23 @@ var getSlideClasses = (spec) => {
     index = spec.index;
   }
 
-  slickCloned = (index < 0) || (index >= spec.slideCount);
+  slickCloned = index < 0 || index >= spec.slideCount;
   if (spec.centerMode) {
     centerOffset = Math.floor(spec.slidesToShow / 2);
-    slickCenter = (spec.currentSlide === index);
-    if ((index > spec.currentSlide - centerOffset - 1) && (index <= spec.currentSlide + centerOffset)) {
+    slickCenter = spec.currentSlide === index;
+    if (index == spec.currentSlide - centerOffset) slickActiveFirst = true;
+    if (index == spec.currentSlide - centerOffset + (spec.slidesToShow-1)) slickActiveLast = true;
+    if (index > spec.currentSlide - centerOffset - 1 && index <= spec.currentSlide + centerOffset) {
       slickActive = true;
     }
   } else {
-    slickActive = (spec.currentSlide <= index) && (index < spec.currentSlide + spec.slidesToShow);
+    slickActive = spec.currentSlide <= index && index < spec.currentSlide + spec.slidesToShow;
   }
-  return classnames({
+  return (0, _classnames2['default'])({
```

Comment:
This is simply `classnames`. It seems you have copied this from a transpiled source. It doesn't work.